### ISSUE TITLE
Flag options not populating

### ISSUE THREAD

**Issue by Cynthia-O** [gradio-app/gradio#221]
Hi! Lovely app, thank you for creating it. 

I'm having trouble getting flagging_options to take. For example...

```
flags = ['this', 'or', 'that']

iface = gr.Interface(fn = myFunction,
inputs = 'text',
outputs = 'text',
allow_flagging = True, 
flagging_options = flags,
flagging_dir = 'flags_from_'+str(datetime.now().date())

iface.launch(share = True)
```

... allows for flagging to register in a log .csv in the desired directory, but no additional columns are created in that log .csv. The .csv contains only a numeric index, an input column, and output column, and a flag column that is populated entirely with NaNs.

**Comment by aliabid94** [comment]
Thanks for pointing this out Cynthia! Should be fixed as of 2.0.6. Will close now, let us know if you have any other issues.

**Comment by Cynthia-O** [comment]
Awesome! Thank you.
